### PR TITLE
Adding governance conditions

### LINKED ISSUE #515: Youtube uploader tool: governance meetings have a title and should only be prepended with the date
In the tool, need to make sure not to rename Governance Committee meetings as they are named already. Just prepend with the date

image

CC: @eddynaka

### LINKED ISSUE #516: Youtube uploader tool: add governance meetings to the playlist automatically
Meetings https://github.com/open-telemetry/community/issues/488 can be added to this playlist automatically: https://www.youtube.com/playlist?list=PLVYDBkQ1Tdyzg1CuQgd9mdjwOUYg7ECYR based on the name. 

CC: @eddynaka

### PR DESCRIPTION
**Pull request by eddynaka**
Fix #515 
Fix #516

### CODE REVIEW

File: tools/OpenTelemetryYoutube/OpenTelemetryYoutube/Program.cs [hunk 1]
```diff
@@ -16,16 +16,22 @@ class Program
     {
         // https://www.youtube.com/channel/UCHZDBZTIfdy94xMjMKz-_MA
         private const string openTelemetryChannelId = "UCHZDBZTIfdy94xMjMKz-_MA";
+        private const string governancePlaylistId = "PLVYDBkQ1Tdyzg1CuQgd9mdjwOUYg7ECYR";
```
Comment (SergeyKanzhelev) [inline tools/OpenTelemetryYoutube/OpenTelemetryYoutube/Program.cs:19]: would be great to have this generalized. Technical committee meetings would be next

**Review by SergeyKanzhelev — APPROVED** [review]
generalization of this logic can be done later

**Comment by SergeyKanzhelev** [comment]
let me know if it's OK to merge

**Comment by eddynaka** [comment]
> let me know if it's OK to merge

You can merge. I will see how the generalization would work.

**Pull request merged**